src/TextInput.js

```diff
@@ -64,6 +66,7 @@ const TextInput = styled(TextInputBase)`
     font-size: ${get('fontSizes.1')}px;
   }
   ${COMMON};
+  ${layout.width}
```

Comment:
You can import `width` directly from `styled-system`
```suggestion
  ${width}
```